addons/web_editor/controllers/main.py

```diff
@@ -76,28 +76,6 @@ def export_icon_to_png(self, icon, color='#000', bg=None, size=100, alpha=255, f
 
             :returns PNG image converted from given font
         """
-        # For custom icons, use the corresponding custom font
-        if icon.isdigit():
-            if int(icon) == 57467:
-                font = "/web/static/fonts/tiktok_only.woff"
-            elif int(icon) == 61593:  # F099
-                icon = "59392"  # E800
-                font = "/web/static/fonts/twitter_x_only.woff"
-            elif int(icon) == 61569:  # F081
-                icon = "59395"  # E803
-                font = "/web/static/fonts/twitter_x_only.woff"
-            elif int(icon) == 59651:
-                font = "/web/static/fonts/bluesky_only.woff"
-            elif int(icon) == 59648:
-                font = "/web/static/fonts/discord_only.woff"
-            elif int(icon) == 59649:
-                font = "/web/static/fonts/google_play_only.woff"
-            elif int(icon) == 59652:
-                font = "/web/static/fonts/kickstarter_only.woff"
-            elif int(icon) == 59650:
-                font = "/web/static/fonts/strava_only.woff"
-            elif int(icon) == 59653:
-                font = "/web/static/fonts/threads_only.woff"
```

Comment:
@Brieuc-brd This was simply removed. It was not to be adapted? So mass_mailing does support `oi` naturally?